lib/atomic_web/live/organization_live/show.ex

```diff
@@ -29,13 +29,16 @@ defmodule AtomicWeb.OrganizationLive.Show do
       }
     ]
 
+    followers = Enum.count(Atomic.Organizations.list_memberships(%{"organization_id" => id}, []))
```

Comment:
```suggestion
    followers_count = Enum.count(Atomic.Organizations.list_memberships(%{"organization_id" => id}, []))
```
`followers` is a misleading name, that could lead someone to think this is a data structure with the organization's followers.